README.md

```diff
@@ -79,3 +79,42 @@ Add `[skip ci]` to the commit subject to skip CI build. Useful for documentation
 
 ### SSH
 Circle CI supports shell access to the build for 120 minutes after the build is finished when the build is started with SSH support. Use "Rerun job with SSH" button in Circle CI UI to start build with SSH support.
+
+## Installation
+
+1. Clone this repository
+
+2. Activate python virtualenv and install extension:
+
+        . /usr/lib/ckan/default/bin/activate
+        cd /usr/lib/ckan/default/src/ckanext-data-qld
+        python setup.py develop
```

Comment:
Why not just clone and install in a single step with `pip install -e 'git+https://github.com/qld-gov-au/ckanext-data-qld'`?

Reply:
Those instructions are generic from the CKAN extension generator. Maybe it's in case a branch switch might be required before installing? Can change if you want me to. Cheers.

Reply:
Eh, no big deal, it would just do away with step 1 and simplify step 2. I suppose the advantage of this approach for a generator is that it doesn't need to specify the repository URL. On the other hand, step 1 should probably mention that you need to clone into a specific location.